input/templates/partials/location/address-list.hbs

```diff
@@ -0,0 +1,32 @@
+<ul class="list">
+  {{#each address}}
+    <li class="item address">
+      <a class="item-link" href="#">
+        <span class="distance">
+          <i class="icon icon-marker"></i>
+          {{this.distance.value}} {{this.distance.unit}}
+        </span>
+
+        <b class="price">
+          {{this.price.value}} {{this.price.currency}}
+        </b>
+
+        <strong class="title">
+          {{this.title}}
+        </strong>
+
+        <div class="box-footer">
+          <div class="status">
+            {{#if this.status}}
```

Comment:
@lauraluiz not sure how we will make it here since we have 3 kinds of status with translations